releasenotes/notes/fix-scheduler-memory-slot-mapping-db299e85aed88037.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
+  - |
+    Fixed basic scheduler such that the scheduler respects MemorySlots
+    in QuantumCircuit measures. With this fix, the measure stimulus pulses will only
+    play on qubits which were explicitly measured, rather than all the qubits in the
+    ``meas_map`` group. (Related to GitHub issue `#2704 <https://github.com/Qiskit/qiskit-terra/issues/2704>`__)
```

Comment:
Is this bug actually included in a release? We shouldn't have a release note for a bug that hasn't been included in a release. The release notes are here for us to document changes since the previous release that we need to to tell our users about.

Reply:
Sorry about the confusion. I'll remove the release note.